File: JAVA/Mahasiswa.java
class Mahasiswa
{
    private String nim;
    private String nama;
    private String prodi;
    private String fakultas;

    Mahasiswa()
    {
        this.nim = "";
        this.nama = "";
        this.prodi = "";
        this.fakultas = "";
    };
    Mahasiswa(String a, String b, String c, String d)
    {
        this.nim = a;
        this.nama = b;
        this.prodi = c;
        this.fakultas = d;
    }

    void setNim(String a)
    {
        this.nim = a;
    }
    void setNama(String a)
    {
        this.nama = a;
    }
    void setProdi(String a)
    {
        this.prodi = a;
    }
    void setFakultas(String a)
    {
        this.fakultas = a;
    }

    String getNim()
    {
        return this.nim;
    }
    String getNama()
    {
        return this.nama;
    }
    String getProdi()
    {
        return this.prodi;
    }
    String getFakultas()
    {
        return this.fakultas;
    }
    
};

File: CPP/mahasiswa.cpp
#include <iostream>
#include <string>

using namespace std;

class mahasiswa
{
private:
    string nim;
    string nama;
    string prodi;
    string fakultas;

public:
    mahasiswa()
    {
        nim = "";
        nama = "";
        prodi = "";
        fakultas = "";
    };
    mahasiswa(string a, string b, string c, string d)
    {
        nim = a;
        nama = b;
        prodi = c;
        fakultas = d;
    }

    void setNim(string a)
    {
        nim = a;
    }
    void setNama(string a)
    {
        nama = a;
    }
    void setProdi(string a)
    {
        prodi = a;
    }
    void setFakultas(string a)
    {
        fakultas = a;
    }

    string getNim()
    {
        return nim;
    }
    string getNama()
    {
        return nama;
    }
    string getProdi()
    {
        return prodi;
    }
    string getFakultas()
    {
        return fakultas;
    }
    ~mahasiswa()
    {
    }
};
